lib/matplotlib/dviread.py

```diff
@@ -583,7 +583,7 @@ class Vf(Dvi):
 
     Parameters
     ----------
-    filename : string or bytestring
```

Comment:
Because it's a path-like?

Reply:
I was adding `path-like`, because that's clearly missing and relevant. Technically, that would be enough (`str` and `bytestring` are `path-like`). However, that's not quite helpful. I don't expect all of our users to know what a `path-like` is, less that a `str` is a `path-like`. The newbie can now still see that a string is fine, and that's the most basic use-case. OTOH, a `bytestring` is already a bit more advanced. Users who know bytestrings probably know path-likes (or at least could guess that if `str` works, `bytestring` will likely too. So `bytestring` is not giving as much additional information that it's worth explicitly mentioning.